tests/integration_tests/scheduler/test_openpbs_driver.py

```diff
@@ -26,3 +30,43 @@ def test_openpbs_driver_with_poly_example():
         "--enable-scheduler",
         "poly.ert",
     )
+
+
+@pytest.mark.timeout(30)
+@pytest.mark.integration_test
+@pytest.mark.usefixtures("copy_poly_case")
+def test_openpbs_driver_with_poly_example_failing_submit_fails_ert_and_propagates_exception_to_user(
+    monkeypatch, caplog
+):
+    monkeypatch.setattr(OpenPBSDriver, "submit", lambda *args, **kwargs: 1 / 0)
```

Comment:
Don't use `1 / 0` to trigger an error in tests. Use `raise RuntimeError` or something where it's obvious that the error is intended by the test.